src/data/routes.json

```diff
@@ -144,7 +144,7 @@
        }
     },
     {
-       "name": "Ascended to Heaven",
```

Comment:
I think ascended is right...

Reply:
Whoops! Must have accidentally deleted that when I copied the routes... fixed it haha